kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -109,10 +166,22 @@ this will then trigger the successful processing of the traveller and put anothe
 }
 ```
 
+* To examine ProcessedTravellers topic and verify it was auto sent
+
+```
+kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic processedtravellers --from-beginning
+```
+
 there are bunch of extension attributes that starts with `kogito` to provide some context of the execution and the event producer.
 
 To take the other path of the process put following message on `travellers` topic
 
+* Send Message to Topic
+```
+kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
+```
+
+With the following content (Cloud Event Format)
 ```
 { 
```

Comment:
This content does not work, it should be full, like the other message sent:

```
{
  "specversion": "0.3",
  "id": "21627e26-31eb-43e7-8343-92a696fd96b1",
  "source": "",
  "type": "TravellersMessageDataEvent_3",
  "time": "2019-10-01T12:02:23.812262+02:00[Europe/Warsaw]",
  "data": { 
	"firstName" : "John", 
	"lastName" : "Doe", 
	"email" : "john.doe@example.com", 
	"nationality" : "American"
	}
}
```

Reply:
and please put the json as a one-line, as the user may want to copy/paste it

Reply:
fixed @radtriste